### PR TITLE
Backport Changes

### PR DESCRIPTION
**Pull request by kayone**

### CODE REVIEW

File: src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs [hunk 1]
```diff
@@ -91,7 +99,7 @@ public List<Series> SearchForNewSeries(string title)
 
                 var httpResponse = _httpClient.Get<List<ShowResource>>(httpRequest);
 
-                return httpResponse.Resource.SelectList(MapSeries);
+                return httpResponse.Resource.SelectList(MapSearhResult);
```
Comment (markus101) [inline src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs]: `MapSearchResult` (missing a `c`)

File: src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs [hunk 2]
```diff
@@ -17,14 +17,16 @@ public class SkyHookProxy : IProvideSeriesInfo, ISearchForNewSeries
     {
         private readonly IHttpClient _httpClient;
         private readonly Logger _logger;
-
+        private readonly ISeriesService _seriesService;
         private readonly IHttpRequestBuilderFactory _requestBuilder;
 
-        public SkyHookProxy(IHttpClient httpClient, ISonarrCloudRequestBuilder requestBuilder, Logger logger)
+        public SkyHookProxy(IHttpClient httpClient, ISonarrCloudRequestBuilder requestBuilder, ISeriesService seriesService, Logger logger)
```
Comment (markus101) [inline src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs]: Should this be split into a service?

File: src/NzbDrone.Common/Serializer/Json.cs [hunk 3]
```diff
@@ -24,22 +26,34 @@ static Json()
...
+                DateTimeZoneHandling = DateTimeZoneHandling.Utc,
+                NullValueHandling = NullValueHandling.Include,
+                Formatting = Formatting.Indented,
+                DefaultValueHandling = DefaultValueHandling.Ignore,
+                ContractResolver = new CamelCasePropertyNamesContractResolver()
+            };
+
+
+            DeserializerSetting.Converters.Add(new StringEnumConverter { CamelCaseText = true });
+            DeserializerSetting.Converters.Add(new VersionConverter());
```
Comment (Taloth) [inline src/NzbDrone.Common/Serializer/Json.cs]: Add DeserializerSetting.Converters.Add(new HttpUriConverter());

or extract adding the converters to a separate method.

**Comment by markus101** [comment]
Doesn't seem to be any traction here, closing.

**Pull request closed without merging**